### PR TITLE
Exporter/Stats/Prometheus: Sanitize metric names

### PR DESCRIPTION
**Pull request by mayurkale22**

### CODE REVIEW

File: packages/opencensus-exporter-prometheus/src/prometheus-stats.ts [hunk 1]
```diff
@@ -158,30 +164,35 @@ export class PrometheusStatsExporter implements StatsEventListener {
    * @param view View used to build the metric name
    */
   private getPrometheusMetricName(view: View): string {
-    return `${this.prefix}_${view.name}_${this.getUnit(view)}`;
+    let metricName;
+    if (this.prefix) {
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply packages/opencensus-exporter-prometheus/src/prometheus-stats.ts]: done, thanks

File: packages/opencensus-exporter-prometheus/src/prometheus-stats.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export class PrometheusStatsExporter implements StatsEventListener {
     port: 9464,
     startServer: false,
     contentType: 'text/plain; text/plain; version=0.0.4; charset=utf-8',
-    prefix: 'opencensus'
+    prefix: ''
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply packages/opencensus-exporter-prometheus/src/prometheus-stats.ts]: It is used while creating metric_name ```getPrometheusMetricName()```.

File: packages/opencensus-exporter-prometheus/src/prometheus-stats.ts [hunk 3]
```diff
@@ -158,30 +164,35 @@ export class PrometheusStatsExporter implements StatsEventListener {
...
-      default:
-        return '';
-    }
+  private sanitizePrometheusMetricName(name: string): string {
+    // replace all characters other than [A-Za-z0-9_].
+    return name.replace(/\W/g, '_');
+  }
+
+  /**
+   * Throws an error labels contains "le" label name in histogram label names.
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply packages/opencensus-exporter-prometheus/src/prometheus-stats.ts]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 - Fix bugs related to Stackdriver Metrics Descriptor and TimeSeries.
 - Add Resource API.
 - Add Metrics API.
+- Remove default prefix from for Prometheus exporter. This could be a breaking change if you have Prometheus metrics from OpenCensus Prometheus exporter of previous versions, please point to the new metrics with no prefix instead.
```
Comment (justindsmith) [inline CHANGELOG.md]: nit: "from for", just pick one of them

Comment (mayurkale22) [reply CHANGELOG.md]: Done, thanks

**Pull request merged**